### PR TITLE
[WIP][Comments Requested] Added more tool qualities and recipes to use them

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
I have created three more tool qualities: CUT_FINE, DRILL_FINE, and GRIP. sharp, fine blades like the scalpel get CUT_FINE, pliers and tongs get GRIP, and the dentist's drill will get DRILL_FINE when it gets added. I haven't added these to recipes yet; do you have any suggestions? (I may be offline for a while, so please don't be upset if I don't respond for a few days).

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
The ````GRIP```` quality sounds like a grand idea, but the others seem like a solution in search of a problem.

**Comment by kevingranade** [comment]
No qualities without uses please.

**Comment by Coolthulhu** [comment]
> No qualities without uses please.

This

Adding qualities should be done when just checking for individual tools causes problems.

**Comment by BorkBorkGoesTheCode** [comment]
Would anybody object if I added a GRIP level 2 requirement to recipes that mold plastic?

**Comment by BorkBorkGoesTheCode** [comment]
This is a work in progress. I am adding new tools and recipes as part of this PR that will make use of the new qualities.

**Comment by BorkBorkGoesTheCode** [comment]
@Rivet-the-Zombie would adding GRIP 2 (pliers) to recipes that use welding torches and lightweight materials be plausible?

**Comment by Rivet-the-Zombie** [comment]
>would adding GRIP 2 (pliers) to recipes that use welding torches and lightweight materials be plausible?

To what purpose? I'm not sure what you're getting at.

**Comment by BorkBorkGoesTheCode** [comment]
I was referring to the recipes that use welding equipment as the primary heat source instead of a forge. Those recipes seem to require some distance between the creator and the metal, but not tongs-level distance.

**Comment by Rivet-the-Zombie** [comment]
Ah right on! Yeah, that would be great. When you're welding in real life, you don't even need anything of the sort in some cases; you can just weld the stuff while it's lying on your work surface without having to hold it.

**Comment by BorkBorkGoesTheCode** [comment]
I think the PR is unmergable due to formatting changes I made while editing the files. It will be a while before I can revert these because of issues with my internet. I will return to editing after the issues are fixed.

File: data/json/recipes/recipe_weapon.json [hunk 1]
```diff
@@ -2916,7 +2914,9 @@
   "decomp_learn": 4,
   "autolearn": false,
   "book_learn": [[ "welding_book", 8 ], ["book_icef", 4]],
-  "qualities" : [ { "id" : "GLARE", "level" : 2 } ],
+  "qualities" : [ 
+      { "id": "GLARE", "level" : 2 },
+      {"id":"GRIP","level": 3,"amount": 1 } ],
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_weapon.json]: Don't add `"amount": 1`. It is redundant.

**Comment by Coolthulhu** [comment]
The GRIP quality isn't a bad idea, but the others should be removed for now.

**Comment by mugling** [comment]
> Ah right on! Yeah, that would be great. When you're welding in real life, you don't even need anything of the sort in some cases; you can just weld the stuff while it's lying on your work surface without having to hold it.

A selection of clamps does improve speed though and when we have optional crafting tools this would be an ideal example

**Comment by BorkBorkGoesTheCode** [comment]
Temporarily closed

**Pull request closed without merging**